Dockerfile

```diff
@@ -157,7 +170,8 @@ ENV AIRFLOW__CORE__HOSTNAME_CALLABLE="socket.gethostname"
 FROM dev_build as prod_build
 
 # Re-declare TARGETARCH for this stage (ARGs don't persist across FROM)
-ARG TARGETARCH
+# Needed for conditional Oracle extras installation below
+ARG TARGETARCH=amd64
```

Comment:
Comment (non-blocking): I suppose we need another check here just like in the `dev_build` stage.

Also, should we _not_ have a default _here_ since the prod image should only be built as part of the CICD which needs to explicitly specify the arch?